exporter/collector.go

```diff
@@ -39,6 +41,7 @@ type JSONMetric struct {
 	LabelsJSONPaths        []string
 	ValueType              prometheus.ValueType
 	EpochTimestampJSONPath string
+	Regex                  *regexp.Regexp
```

Comment:
This should be `IncludeRegex` in case we want to also have an `ExcludeRegex` for both positive and negative matching.

Reply:
Do we really need ExcludeRegex?
Regex support both positive and negative matching

Reply:
changed as you wish